crates/ursa-index-provider/src/provider.rs

```diff
@@ -181,9 +181,10 @@ where
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Message {
     pub Cid: Cid,
-    pub Addrs: Vec<Multiaddr>,
+    pub Addresses: Vec<Multiaddr>,
```

Comment:
Is the announcement message struct changed from the indexer?
> https://docs.cid.contact/index-provider/overview
```go
{
	Cid:      cid.Cid
	Addrs     [][]byte
	ExtraData []byte `json:",omitempty"`
}
```

Reply:
You are right, reverting, it irks me